app/src/main/java/fr/free/nrw/commons/nearby/model/ResultTuple.kt

```diff
@@ -1,15 +1,21 @@
 package fr.free.nrw.commons.nearby.model
 
+import com.google.gson.annotations.SerializedName
+
 class ResultTuple {
+    @SerializedName("xml:lang")
+    val lang: String
```

Comment:
Would you mind calling it `language` everywhere? Even if the server returns it as `lang`, on the app side we should use our convention which is to use more explicit names. Thanks!